content/acra/configuring-maintaining/general-configuration/acra-addzone.md

```diff
@@ -224,42 +224,142 @@ Should be provided only with `--keystore_encryption_type=<kms_encrypted_master_k
 {{< /hint >}}
 
 
-### HashiCorp Vault
-
-`acra-addzone` can read `ACRA_MASTER_KEY` from HashiCorp Vault instead of environment variable.
+### Hashicorp Vault
 
 * `--vault_connection_api_string=<url>`
 
-  Connection string (like `https://example.com:8200`) for connecting to HashiCorp Vault.
-  If not specified, `ACRA_MASTER_KEY` environment variable will be used.
+  Connection string (like `http://x.x.x.x:yyyy`) for loading `ACRA_MASTER_KEY` from HashiCorp Vault.
+  Default is empty (`ACRA_MASTER_KEY` environment variable is expected).
 
-* `--vault_secrets_path=<kv-path>`
+* `--vault_secrets_path=<path>`
 
-  Path to KV Secrets directory in Vault used to store `ACRA_MASTER_KEY`.
+  KV Secret Path for reading `ACRA_MASTER_KEY` from HashiCorp Vault.
   Default is `secret/`.
 
-* `--vault_tls_ca_path=<path>`
```

Comment:
we should leave it too due to keeping backward compatibility, as commented in related pr